internal/cli/analytics.go

```diff
@@ -21,7 +23,7 @@ const tiltAppName = "tilt"
 const disableAnalyticsEnvVar = "TILT_DISABLE_ANALYTICS"
 const analyticsURLEnvVar = "TILT_ANALYTICS_URL"
 
-var analyticsService analytics.Analytics
+var analyticsService *tiltanalytics.TiltAnalytics
```

Comment:
ughhhhh changing this file, and it being untested, makes me antsy. i suppose we'll have to do it soon?? but like, i'm a little inclined to do it when we pull the trigger on newAnalytics so that we don't need to worry about maintaining the old flow and the new flow side by side? 🤷‍♀ 😬